pandas/core/generic.py

```diff
@@ -5716,24 +5716,75 @@ def clip_upper(self, threshold, axis=None, inplace=False):
 
     def clip_lower(self, threshold, axis=None, inplace=False):
         """
-        Return copy of the input with values below given value(s) truncated.
+        Trim the caller's values below a given `threshold`.
+
+        Elements below the `threshold` will be changed to match the
+        `threshold` value(s).
 
         Parameters
         ----------
-        threshold : float or array_like
-        axis : int or string axis name, optional
+        threshold : float or array-like
+            Lower value(s) to which the input value(s) will be trimmed.
+        axis : {0 or 'index', 1 or 'columns', None}, default None
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
 
-        See Also
-        --------
-        clip
+            .. versionadded:: 0.21.0
 
         Returns
         -------
-        clipped : same type as input
+        Series or DataFrame
+            Same type as caller.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'a': [0.740518, 0.450228, 0.710404, -0.771225],
+        ...                    'b': [0.040507, -0.45121, 0.760925, 0.010624]})
```

Comment:
Personally I find the data used in #20212 easier to understand. It requires a decent amount of concentration to see what the examples are doing with so many decimal numbers.